src/pages/_middleware.tsx

```diff
@@ -0,0 +1,43 @@
+import { NextResponse } from "next/server";
```

Comment:
nit: `_middleware.ts` work? I don't think this can ever have JSX

Reply:
It does work as a .ts, to my knowledge.